### DISCUSSION TITLE
Transitive vulnerable dependency in version 3.0.0

### DISCUSSION THREAD

**Discussion by ydrozhdzhal** [jxlsteam/jxls#325] [Q&A]
Hi

We migrated our project to library version 3.0.0 and it still contains vulnerable transitive dependency **commons-collections:commons-collections:3.2.2** (resolved through **commons-beanutils:commons-beanutils:1.9.4**)

Can you exclude this vulnerable dependency from next minor release?

More details: https://devhub.checkmarx.com/cve-details/Cx78f40514-81ff/

**Comment by SoltauFintel** [comment]
Discussions is the wrong area. Please send us a pull request.

**Reply by ydrozhdzhal** [reply]
Sorry, but will not do that. Will create issue for this. In case this will be not fixed in next releases then I prefer to switch to other library.

**Reply by SoltauFintel** [reply]
An issue is also ok.
